vendor/assets/stylesheets/rolodex/components/_forms.sass

```diff
@@ -31,7 +31,8 @@ input[type="search"]
   box-sizing: border-box
 
 input[type="search"]::-webkit-search-cancel-button,
-input[type="search"]::-webkit-search-decoration
+input[type="search"]::-webkit-search-decoration,
+input[type="submit"]
   -webkit-appearance: none
```

Comment:
If you add `-webkit` to any property it will be skipped by Autoprefixer.

If the goal is to only have the `-webkit` prefix and none other you're good with the above.

For reference, you can play with Autoprefixer in the browser at https://autoprefixer.github.io/.